easybuild/easyconfigs/b/bzip2/bzip2-1.0.8-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,28 @@
+name = 'bzip2'
+version = '1.0.8'
+
+homepage = 'https://sourceware.org/bzip2'
+description = """
+ bzip2 is a freely available, patent free, high-quality data compressor. It
+ typically compresses files to within 10% to 15% of the best available
+ techniques (the PPM family of statistical compressors), whilst being around
+ twice as fast at compression and six times faster at decompression.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://sourceware.org/pub/%(name)s/']
+sources = [SOURCE_TAR_GZ]
+patches = ['bzip2-1.0.6-pkgconfig.patch']
```

Comment:
We should fix the patch. See https://github.com/easybuilders/easybuild-easyconfigs/issues/13513

Reply:
Thank you! I wasn't aware of this issue. There is no open PR yet for the updated patch, right?